### ISSUE TITLE
Stuck at BH 10362082

### ISSUE THREAD

**Issue by tomazcz** [iconation/aegis#1]
Hey Spl3en, getting this error now:
```
Aegis: /root/IST/Aegis2/Aegis/third_party/nlohmann/json.hpp:3331: const value_type& nlohmann::basic_json<ObjectType, ArrayType, StringType, BooleanType, NumberIntegerType, NumberUnsignedType, NumberFloatType, AllocatorType, JSONSerializer>::operator[](T*) const [with T = const char; ObjectType = std::map; ArrayType = std::vector; StringType = std::__cxx11::basic_string<char>; BooleanType = bool; NumberIntegerType = long int; NumberUnsignedType = long unsigned int; NumberFloatType = double; AllocatorType = std::allocator; JSONSerializer = nlohmann::adl_serializer; nlohmann::basic_json<ObjectType, ArrayType, StringType, BooleanType, NumberIntegerType, NumberUnsignedType, NumberFloatType, AllocatorType, JSONSerializer>::const_reference = const nlohmann::basic_json<>&; nlohmann::basic_json<ObjectType, ArrayType, StringType, BooleanType, NumberIntegerType, NumberUnsignedType, NumberFloatType, AllocatorType, JSONSerializer>::value_type = nlohmann::basic_json<>]: Assertion `m_value.object->find(key) != m_value.object->end()' failed.
Aborted (core dumped)
```

**Comment by Spl3en** [comment]
Hey, thanks for reporting it. I confirm I have the same behavior here, I'm investigating

**Comment by Spl3en** [comment]
Fixed in v1.3 : https://github.com/iconation/Aegis/releases/tag/v1.3

Thanks for the report
